README.md

```diff
@@ -11,37 +11,77 @@ intuitive, and predictable.
 
 ```console
 admin              Administrative commands
-alloc              allocate a new instance for interactive use
+alloc              Interactively enqueue a job within a sub allocation
 attach             Interactively attach to job
+batch              Non-interactively enqueue a job within a sub allocation
 cancel             Cancel one or more jobs
-info               Display info for a job
-kill               Send signal to one or more running jobs
+info               Display info for one or more jobs
+kill               Forcefully kill one or more running jobs
 signal             Send a signal (e.g., raise exception) on one or more jobs
 cron               Schedule tasks on timers and events
-exec               Execute processes across flux ranks
-jobs               flux jobs queue listing
-purge              Purge the oldest inactive jobs
-resource           list/manipulate Flux resource status
-run                run a job interactively
+exec               Execute processes across flux ranks in a job
+jobs               Show queued, running, and historical flux jobs
+prune              Purge the oldest inactive jobs
+run                Run a command as a job interactively
 ssh                Create proxy environment for Flux instance
-submit             enqueue a job
-system             list/manipulate Flux resource status
+submit             Non-interactively enqueue a job
+system             System commands
 version            Display flux version information
-wait               Wait for job(s) to complete.   
+wait               Wait for job(s) to complete  
 ```
 
 ### flux submit
 
-Submit flux jobs
+#### Interactively
+
+Run a flux job in an existing flux allocation
+```console
+# Submit a job directly to get back an id
+$ flux run sleep 10
+ƒhejTu4KTFm
+
+# Submit a flux job file
+$ flux run -f mybatch.sh
+ƒhejTu4KTFm
+```
+
+Run a flux job in a new flux allocation
+
+```console
+# Submit a job directly to get back an id
+$ flux alloc sleep 10
+ƒhejTu4KTFm
+
+# Submit a flux job file
```

Comment:
I would put this example under the first bullet and delete this comment, and then we are missing a `flux alloc` without a command to show getting the interactive allocation.

Reply:
Updated. Let me know if this makes more sense.

Reply:
heyo! Could you not unresolve next time if you prompt a response? I just searched a few minutes looking for this and couldn't find it because it was resolved.

Reply:
And yep looks good (and now is when I would resolve this thread, if it weren't resolved.)